solutions/src/main/java/ru/sberbank/school/task02/ClientExceptionTests.java

```diff
@@ -0,0 +1,36 @@
+package ru.sberbank.school.task02;
+
+import org.junit.Before;
+import org.junit.Test;
+import ru.sberbank.school.task02.util.ExternalQuotesServiceDemo;
+import ru.sberbank.school.task02.util.FxRequest;
+
+
+public class ClientExceptionTests {
+    static private FxConversionService converter;
```

Comment:
&#39;private&#39; modifier out of order with the JLS suggestions.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1370544565>*